controllers/openstackdataplanedeployment_controller.go

```diff
@@ -147,6 +147,21 @@ func (r *OpenStackDataPlaneDeploymentReconciler) Reconcile(ctx context.Context,
 		nodeSets.Items = append(nodeSets.Items, *nodeSetInstance)
 	}
 
+    // Add Ansible variables from the OpenStackDataPlaneDeployment to the AnsibleEESpec
+    ansibleEESpec.AnsibleVars = make(map[string]string)
```

Comment:
ansibleEESpec is listed as undefined here and `ansibleEESpec.AnsibleVars` isn't consumed anywhere

Reply:
You can use the ExtraVars field on ansibleEEspec (see https://github.com/openstack-k8s-operators/openstack-ansibleee-operator/blob/main/api/v1alpha1/openstack_ansibleee_types.go#L119C1-L119C67), then do something like:

ansibleEESpec.ExtraVars = Deployment.Spec.ansibleExtraVars